account_tax_analysis/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Account Tax analysis
+====================
+
+This add-on is a must if you want to be able to validate your VAT form.
+
+Thanks to a new menu 'Invoicing / Reports / Taxes Analysis'
+you are able to group accounting entries by Taxes (VAT codes)
+and/or financial accounts.
+
+This way you will find easily differences you may see between
+the Odoo tax report and what you see in your books.
+
+Installation
+============
+
+Just install it
+
+Configuration
+=============
+
+Nothing
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/92/10.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/account-financial-tools/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Vincent Renaville (Camptocamp SA) <vincent.renaville@camptocamp.com>
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
```

Comment:
can you use the new `readme` [fragments](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst), in order to get the autogenerated README.rst?